tests/test_datetime/conftest.py

```diff
@@ -89,3 +89,43 @@ def df_datetime_transformed(df_datetime):
     }
     df = df_datetime.join(pd.DataFrame(data))
     return df
+
+
+@pytest.fixture(scope="module")
```

Comment:
this df is the expected output if I understand correctly, then it should go directly in the test file. In the conftest we put intput dataframes